builder/js/ncf.js

```diff
@@ -507,17 +509,18 @@ app.controller('ncf-builder', function ($scope, $modal, $http, $log, $location,
     if (method_call.method_name in $scope.generic_methods ) {
       var method = $scope.generic_methods[method_call.method_name];
       for (var i = 0; i < method.parameter.length; i++) {
-         var param_name = method.parameter[i].name;
+         var parameter = method.parameter[i];
          var param_value = method_call.args[i];
          // Maybe parameter does not exists in current method_call, replace with empty value
          param_value = param_value !== undefined ? param_value : '';
-         var parameter = createParameter(param_name,param_value);
+         parameter["value"] = param_value;
          params.push(parameter);
       }
     } else {
       // We have no informations about this generic method, just call it 'parameter'
       params = method_call.args.map( function(arg) { return createParameter ('parameter', arg);});
     }
+    console.log(params)
```

Comment:
Removed in https://github.com/Normation/ncf/commit/0ef6c5c352b0710bd411bebeb122c9aec3b1e259 !! Thanks @ncharles